generated/ruby/google-cloud-ruby/google-cloud-trace/google-cloud-trace-v1.gemspec

```diff
@@ -2,7 +2,7 @@
 # encoding: utf-8
 
 Gem::Specification.new do |s|
-  s.name          = 'google-cloud-trace-v1'
+  s.name          = 'stackdriver-trace-v1'
```

Comment:
I think the gem name should still be google-cloud-trace-v1, just description should be 'Stackdriver Trace'.